pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Including this file in the project makes it possible to refer to the project from `deps.edn`, e.g. `io.lambdaforge/wanderung {:git/url "https://github.com/jonasseglare/wanderung" :sha "fe35e0282f787d7f519acc9201ee99eb3eb881ec"}}`.